sdk/src/main/java/ly/count/android/sdk/ModuleFeedback.java

```diff
@@ -448,6 +447,8 @@ void reportFeedbackWidgetManuallyInternal(@Nullable CountlyFeedbackWidget widget
 
         if (widgetResult != null) {
             //removing broken values first
+            UtilsInternalLimits.removeUnsupportedDataTypes(widgetResult);
```

Comment:
what would this do if it comes across a key value pair where the value is an array?

Reply:
it will remove the array

Reply:
But this function is not used in user profiles, because client could send array types If you were intended to ask that question

Reply:
I am curious if this would remove some results of a widget that the person is trying to report

Reply:
It is possible if that is the case. However, it was like that before. So no one reported any suspicious thing. But to make sure maybe we could remove it from here

Reply:
can you check out here and let us know if we were somehow damaging their widget results even before: https://support.count.ly/hc/en-us/articles/9290669873305-A-Deeper-Look-at-SDK-concepts#h_01HABT18WTFWFNKVPJJ6G6DEM4 
<3

Reply:
I have checked it, and we were not. Even multiple choices are reported as concatenated string

Reply:
then all cool